app/javascript/packs/stylesheets/bootstrap.scss

```diff
@@ -195,10 +118,64 @@ select.form-control {
   font-weight: 250 !important;
 }
 
+.nav-tabs .nav-link {
+  border-top-left-radius: 0 !important;
+  border-top-right-radius: 0 !important;
+}
+
+.nav-tabs .nav-link:hover {
+  border-color:#b5b5b5;
+  background-color:white;
+}
+
 .nav-link-inactive {
   color: rgba(255, 255, 255, 0.8) !important;
 }
 
+.nav-item:last-child {
+  margin-left: auto;
+}
+
+/* styling for navbar on smaller screens only */
+@media only screen and (max-width : 1310px) {
+  .nav-tabs .nav-link {
+    border: 1px #dee2e6 solid;
+    margin-bottom: -.05px;
+  }
+
+  .nav-tabs .nav-link.exposure {
+    width: 190px;
+  }
+
+  .nav-tabs .nav-link.isolation {
+    width: 265px;
+  }
```

Comment:
I swear 265px worked for me before but now it looks like it needs to be 266px

![image](https://user-images.githubusercontent.com/8235974/121903903-bf64af80-ccf6-11eb-8064-634634e8612d.png)

Reply:
It works for me??

![Screen Shot 2021-06-14 at 10 10 58 AM](https://user-images.githubusercontent.com/35042815/121906160-d4dad900-ccf8-11eb-83a7-3fb0889b3208.png)

Reply:
That would be because my count (140) is wider than yours (30). Is there a way to dynamically determine the tab width, basing on the maximum auto width (if you know what I mean)? If not, we're just gonna have to build in some extra buffer space. There's no hard limit to what the count could be (though realistically I guess it would be 5 digits)

Reply:
I'll build in extra buffer space, again that would be too complicated because this is now located in the css file which is detached from where the counts live

Reply:
That's the downside to having a fixed width instead of the auto width...

Reply:
fixed here: https://github.com/SaraAlert/SaraAlert/pull/931/commits/69c3e7f6dba9e9b0effde746c6be08a6a2362a05